### PR TITLE
Yet Another Ovulo Bugfix

### PR DESCRIPTION
**Pull request by DatBoiTim**
## About The Pull Request
I have learned a great many things. Including not coding while at work on a Mac, in the middle of a semester.

Add initial reagents just doesn't work. So that's cool. Ovulogenesis doesn't use that anymore. Now they have the reagents.

Eggsplosion didn't work because for some fucking reason the disease stat for transmission is named "transmittable" in code, while the symptom and screen reference to it is transmission. Due to this the threshold was checking a stat that didn't "exist".

## Why It's Good For The Game
FIX BUG

## Changelog
:cl: DatBoiTim
fix: Ovulogenesis uses a new method for adding its reagents now they all get added
fix: Eggsacs actually have their transmission thresholds work
/:cl:

### CODE REVIEW

**Comment by DatBoiTim** [comment]
This PR has sparked me to open #4192 because of the stupid shit with the assoc list naming

File: code/datums/diseases/advance/symptoms/skin.dm [hunk 1]
```diff
@@ -207,10 +207,10 @@ BONUS
...
 	symptom_delay_min = 60
 	symptom_delay_max = 105
-	var/big_heal
-	var/all_disease
-	var/eggsplosion
-	var/sneaky
+	var/big_heal = FALSE
+	var/all_disease = FALSE
+	var/eggsplosion = FALSE
+	var/sneaky = FALSE
```
Comment (Archanial) [inline code/datums/diseases/advance/symptoms/skin.dm]: does this change anything? if not, why change it?

File: code/datums/diseases/advance/symptoms/skin.dm [hunk 2]
```diff
@@ -267,6 +267,7 @@ BONUS
 	icon = 'icons/obj/food/food.dmi'
 	icon_state = "eggsac"
 	bitesize = 4
+	var/eggsplode_delay = 100 SECONDS
```
Comment (Archanial) [inline code/datums/diseases/advance/symptoms/skin.dm]: make it define instead

File: code/datums/diseases/advance/symptoms/skin.dm (hunk 1 above)
Comment (DatBoiTim) [reply code/datums/diseases/advance/symptoms/skin.dm]: Side effect of me really trying to figure out why the thresholds weren't working. I was VVing the symptom and noticed they were null despite the stats supposed to be activating them

Comment (Archanial) [reply code/datums/diseases/advance/symptoms/skin.dm]: If you don't initialize them with a value they will be null, which acts like false

Comment (DatBoiTim) [reply code/datums/diseases/advance/symptoms/skin.dm]: I am aware it was more a thing of I set a switchboard sorta thing during diagnostics.

Comment (DatBoiTim) [reply code/datums/diseases/advance/symptoms/skin.dm]: Doesn't matter I've cleaned that up

File: code/datums/diseases/advance/symptoms/skin.dm (hunk 2 above)
Comment (DatBoiTim) [reply code/datums/diseases/advance/symptoms/skin.dm]: b96a0b8

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**